### ISSUE TITLE
DatabaseMigrations sample (dotnet ef migrations)

### ISSUE THREAD

**Issue by Fazer01** [dotnet/aspire-samples#151]
Hi there, 

Playing around through the `DatabaseMigrations` sample, when hitting F5 everything works fine. The SQL server is spun up in a container because local development and the `Api` project is adding entries to the dabase pointed to by `db1` connection when hitting the  `/` endpoint of the service. :) So far so good. 

After running the sample I executed the steps to add a `Name` property to the `Entry` entity and run the command `dotnet ef migrations add MyNewMigrationAddNameToEntry --project ..\DatabaseMigrations.ApiModel\DatabaseMigrations.ApiModel.csproj` But when I run that command, it says that the `ConnectionString:db1` is not known which is correct, because there is no fixed connectionstring as I develop locally. Do I need to specify the connectionstring `db1` in my local development app settings? If so, how do I do that as I want to make use of the sql server container during local development...

Errormessage in the console when running the `dotnet ef` command as mentioned above.: 
**The exception 'ConnectionString is missing. It should be provided in 'ConnectionStrings:db1' or under the 'ConnectionString' key in 'Aspire:Microsoft:EntityFrameworkCore:SqlServer' or 'Aspire:Microsoft:EntityFrameworkCore:SqlServer:MyDb1Context' configuration section.' was thrown while attempting to find 'DbContext' types. For the different patterns supported at design time, see https://go.microsoft.com/fwlink/?linkid=851728**

TL;DR: 
How to use the local development sql server container (which is destroyed when the application stops) with `dotnet ef migrations` using .NET Aspire?

Thanks in advance,
Regards, 

Tom

**Comment by adamtrip** [comment]
Hey @Fazer01 

If I understood correctly you want to persist your database when using Aspire created SQL Server docker containers? If so check Volume Mounts example: https://github.com/dotnet/aspire-samples/blob/main/samples/VolumeMount/VolumeMount.AppHost/Program.cs

**On your connection string:** Because how ef tools works, you need to have the connection string defined on your `appSettings.json`. It can be a dummy connection string if you just plan to create migrations but for removing them, since it needs to connect to the db, you need to put the real connection.

However, check the https://github.com/dotnet/aspire-samples/tree/main/samples/DatabaseMigrations sample because changes have been made on how to achieve this

**Comment by DamianEdwards** [comment]
@JamesNK

**Comment by DamianEdwards** [comment]
@Fazer01 I'm not able to reproduce your issue. If I follow the steps in the README I can successfully create the migration. Can you run `dotnet ef --version` to verify which version of the EF tool you're using?

**Comment by DamianEdwards** [comment]
Closing this out as I can't repro and it's over a month old. Please re-open if there's more information.

**Comment by Fazer01** [comment]
Hi Damian. I have awaited the GA of .NET Aspire. As it turns out, the migration service (which is part of the [.NET Aspire docs page](https://learn.microsoft.com/en-us/dotnet/aspire/database/ef-core-migrations)) works like a charm! I'm sorry for not replying earlier! 

Regards, 

Tom
